lambdas/death-enrichment/src/test/java/uk/gov/di/data/lep/GroDeathEnrichmentTest.java

```diff
@@ -60,27 +59,19 @@ void enrichGroDeathEventDataReturnsEnrichedData() throws JsonProcessingException
         sqsMessage.setBody("A message body");
         var sqsEvent = new SQSEvent();
         sqsEvent.setRecords(List.of(sqsMessage));
+        var groJsonRecord = new GroJsonRecordBuilder().build();
+        var deathNotificationSet = mock(DeathNotificationSet.class);
 
         when(objectMapper.readValue(sqsMessage.getBody(), GroJsonRecord.class))
-            .thenReturn(new GroJsonRecordBuilder().build());
+            .thenReturn(groJsonRecord);
+        var deathNotificationSetMapper = mockStatic(DeathNotificationSetMapper.class);
```

Comment:
Given this is static and we expect it to work (we've covered it in tests etc) is there an advantage in mocking it, rather than just making a few specific assertions that the mapped response looks like we expect it to?

Reply:
I would rather not test implementation in the higher tests, so leaving it mocked makes sense (also mean we don't have to generate the whole object)